app/lib/api/reactQuery/hooks/query/users.ts

```diff
@@ -0,0 +1,16 @@
+import { useQuery } from '@tanstack/react-query';
+import queryClient from '../../queryClient';
+import QUERY_KEYS from '../../queryKeys';
+import API from '../../../apiWrapper';
+
+const useUserInfo = (webId?: string) => useQuery(
+    QUERY_KEYS.userInfo(webId),
+    () => API.users.getUserInfo(webId || ''),
+    {
+        ...queryClient.getDefaultOptions(),
+        enabled: webId !== undefined,
+        staleTime: 60 * 1000, // long stale time, user info should never really change
```

Comment:
Hmm yeah, I thought it was longer already. It can be much longer than a minute, I'll change it to an hour, same with map info. That shouldn't change at all either.

And yes, you can invalidate the cache of any query using: `queryClient.invalidateQueries(QUERY_KEYS.userInfo(webId))`. Doing it in the way you suggest would be difficult, since you need to invalidate it on the front-end, and we don't really know anything about changes unless we perform a request.

Reply:
Well, I meant a situation where we just changed something via the UI so we know that we have to re-fetch the data. But it's not like the user has much to change anyway.